src/SemiclassicalOrthogonalPolynomials.jl

```diff
@@ -509,6 +512,79 @@ function LazyArrays.cache_filldata!(P::SemiclassicalJacobiFamily, inds::Abstract
     P
 end
 
+###
+# here we construct hierarchies of c weight sums by means of contiguous recurrence relations
+###
+
+mutable struct SemiclassicalJacobiCWeightFamily{T, C} <: AbstractVector{SemiclassicalJacobiWeight{T}}
```

Comment:
Add a docstring: I have no idea what this is meant to be doing.

Also why is it mutable?

Reply:
added a docstring, though it is intended to be an internal method that picks up on a computation important for annulus applications, not something that a user manually constructs. it is automatically used when broadcasting weight functions over the right kind of input. Won't argue with more docs though

Reply:
the point is really that sum.(::SemiclassicalJacobiCWeightFamily) is fast and uses recurrence instead of individual sums via pFq